### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by CEKrause**
Update repo README with some additional text, installation instructions and citation information.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -19,19 +19,35 @@ Digital Earth Australia Waterbodies
...
 
-Up to date information about the extent and location of surface water provides all Australians with a common understanding of this valuable and increasingly scarce resource.
+**Citing Digital Earth Australia Waterbodies:**
 
-Digital Earth Australia Waterbodies shows the wet surface area of waterbodies as estimated from satellites. It does not show depth, volume, purpose of the waterbody, nor the source of the water.
+    Krause, Claire E.; Newey, Vanessa; Alger, Matthew J.; Lymburner, Leo. 2021. "Mapping and Monitoring the Multi-Decadal Dynamics of Australia’s Open Waterbodies Using Landsat" Remote Sens. 13, no. 8: 1437. https://doi.org/10.3390/rs13081437
 
-Digital Earth Australia Waterbodies uses Geoscience Australia’s archive of over 30 years of Landsat satellite imagery to identify where almost 300,000 waterbodies are in the Australian landscape and tells us the wet surface area within those waterbodies.
+----------
```
Comment (MatthewJA) [inline README.rst:28]: Please make this a subheading!

Comment (MatthewJA) [reply README.rst:28]:
```
heading
---------
```

**Review by MatthewJA — CHANGES_REQUESTED** [review]
The two images should be moved to `figures/` not `Documentation_figures/`. Besides that and my other comments, looks good. Thanks for this.

File: README.rst (hunk 1 above)
Comment (CEKrause) [reply README.rst:28]: Leaving as a horizontal line.

**Comment by CEKrause** [comment]
Documentation_figures folder is removed (no idea where it came from)

**Review by MatthewJA — APPROVED** [review]
LGTM!

**Pull request merged**